backend/server/controllers/participants.js

```diff
@@ -9,10 +9,45 @@ module.exports = {
                 message: 'The attribute "experiment_id" of an instance "Participant" cannot be empty.'
           });
         }
+        if (!req.body.name) {
+            return res.status(400).send({
+                status: 400,
+                message: 'The attribute "name" of an instance of "Experiment" cannot be empty.'
```

Comment:
This is not an instance of an Experiment. Please, see other similar mistakes below.